mmseg/__init__.py

```diff
@@ -58,17 +58,23 @@ def digit_version(version_str: str, length: int = 4):
 mmcv_version = digit_version(mmcv.__version__)
 
 
-assert (mmcv_min_version <= mmcv_version < mmcv_max_version), \
```

Comment:
maybe the following can better solve the problem

source:
min<=current<max

modified:
min<=current<=max